frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/TaskStatus.java

```diff
@@ -131,6 +132,14 @@ public void setContainerIp(String containerIp) {
     this.containerIp = containerIp;
   }
 
+  public String getContainerPorts() {
+    return containerPorts;
+  }
+
+  public void setContainerPorts(String ports) {
+    containerPorts = ports;
+  }
```

Comment:
Can we align the input Port style with output

Reply:
Such as make portRange to be string in ResourceDescriptor

---
In reply to: [167152760](https://github.com/Microsoft/pai/pull/318#discussion_r167152760) [](ancestors = 167152760)

Reply:
No, PortRange in ResourceDescriptor need do calculation.

---
In reply to: [167156460](https://github.com/Microsoft/pai/pull/318#discussion_r167156460) [](ancestors = 167156460,167152760)